iothub/service/samples/solutions/DigitalTwinClientSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -65,33 +65,34 @@ private async Task UpdateDigitalTwinComponentPropertyAsync()
             int desiredTargetTemperature = s_random.Next(0, 100);
 
             const string targetTemperaturePropertyName = "targetTemperature";
-            var updateOperation = new UpdateOperationsUtility();
+            var updateOperation = new JsonPatchDocument();
```

Comment:
@timtay-microsoft why not make the update API take this type rather than string? Do we not want to rely on an external library (maybe we already link to it?)?

Reply:
I don't know how intentional this was when we originally created the v1 digital twins client. I suspect we took it as a string to allow users to construct this field however they want rather than forcing them to use the UpdateOperationsUtility/JsonPatchDocument type

Reply:
Fair. Do you think we should consider taking this type directly, like we do in the ADT client? Or just reference in the doc comments that it can be useful and demonstrate it with samples?